### DISCUSSION TITLE
Ready to take back the project :-)

### DISCUSSION THREAD

**Discussion by carstengehling** [jirastopwatch/jirastopwatch#90] [General]
Hi @norwd, Carsten Gehling here. I am the original author of Jira StopWatch. I can see, that there generally is trouble finding a new maintainer. When I asked to have new maintainers, it was because I no longer needed to register time through Jira anymore, so my own motivation for continuing the development shrinked.

Now I am back in a position, where it makes sense to develop on Jira StopWatch. And I also know a lot of people like the program, so I hate to see it whither away. So if you are interested, I will readily take back the ownership of Jira StopWatch.

You can contact me on carsten@sarum.dk for more details.

**Comment by norwd** [comment]
I've send you an email and an org invite @carstengehling :-)
